tools/staramr/staramr_search.xml

```diff
@@ -38,33 +39,36 @@
             --no-exclude-genes
         #end if
 
+        #if str($advanced.plasmidfinder_type.plasmidfinder_type_condition) != 'default'
+            --plasmidfinder-database-type '${advanced.plasmidfinder_type.plasmidfinder_type_condition}'
+        #end if
+
         --output-summary $summary
+        --output-detailed-summary $detailed_summary
         --output-resfinder $resfinder
-        #if $use_pointfinder.enable
+        --output-plasmidfinder $plasmidfinder
+        #if str($use_pointfinder.use_pointfinder_condition) != 'default'
             --output-pointfinder $pointfinder
         #end if
         --output-settings $settings
         --output-excel $excel_proper_extension
 
         --output-hits-dir staramr_hits
 
-        #if $use_pointfinder.enable
-            --pointfinder-organism $use_pointfinder.organism
+        #if str($use_pointfinder.use_pointfinder_condition) != 'default'
+            --pointfinder-organism '${use_pointfinder.use_pointfinder_condition}'
         #end if
 
         $named_genomes
     ]]></command>
     <inputs>
         <param type="data" name="genomes" format="fasta" multiple="true"/>
         <conditional name="use_pointfinder">
-            <param name="enable" type="boolean" label="Enable scanning for point mutations using the PointFinder database" />
-            <when value="true">
-                <param name="organism" type="select" label="Select a pointfinder organism">
-                    <option value="salmonella" selected="true">Salmonella</option>
-                    <option value="campylobacter">Campylobacter</option>
-                </param>
-            </when>
-            <when value="false" />
+            <param name="use_pointfinder_condition" type="select" label="Enable scanning for point mutations using the PointFinder database">
+                <option value="default" selected="true">Disable PointFinder</option>
```

Comment:
Could you change the default value here to something more descriptive as to what default condition is (e.g., `value="disabled"`)?

Reply:
Fixed in my next commit